### PR TITLE
DOC-812: Add release note for keyboard control of editor resizing

### PR DESCRIPTION
**Pull request by ashfordneil**
Related Ticket: DOC-812

Description of Changes:
* Add a release note to describe how the resize handle can be controlled with the keyboard

Pre-checks:
- [x] Branch prefixed with `feature/` or `hotfix/`
- [x] ~`_data/nav.yml` has been updated (if applicable)~
- [x] ~Files has been included where required (if applicable)~
- [x] ~Files removed have been deleted, not just excluded from the build (if applicable)~
- [x] (New product features only) Release Note added

Review:
- [x] Documentation Team Lead has reviewed
- [x] Product Manager has reviewed

### CODE REVIEW

File: release-notes/release-notes58.md [hunk 1]
```diff
@@ -27,6 +27,12 @@ The following new features were added for the {{site.productname}} 5.8 release.
 
 ### Feature name
 
+### Added support for resizing the editor using the keyboard
+
+Keyboard navigation can now be used to select the editor resize handle. Once it is selected, the arrow keys can be used to resize the editor.
+
+For more information on editor resizing, see [Resize]({{site.baseurl}}/configure/editor-appearance#resize).
```
Comment (tylerkelly13) [inline release-notes/release-notes58.md:34]:
```suggestion
For information on editor resizing, see: [Resize]({{site.baseurl}}/configure/editor-appearance#resize).
```
No More! :stuck_out_tongue:

**Review by tylerkelly13 — APPROVED** [review]

**Review by metricjs — APPROVED** [review]
Ticking but, do we care to describe how the arrow keys control resizing? I guess it's the obvious controls though so probably not required...

**Review by lnewson — APPROVED** [review]

**Comment by JamesToohey** [comment]
> Ticking but, do we care to describe how the arrow keys control resizing? I guess it's the obvious controls though so probably not required...

I agree with this, I think we should document the direction, most importantly up/down

**Review by EkimChau — APPROVED** [review]

File: _includes/configuration/resize.md [hunk 2]
```diff
@@ -2,6 +2,11 @@
 
 This option gives you the ability to disable the resize handle or set it to resize both horizontal and vertically. The option can be true, false or the string `'both'`. `False` disables the resize, `true` enables vertical resizing only, `'both'` makes it possible to resize in both directions horizontal and vertical.
 
+When resizing is enabled, the editor can be resized by either:
+
+* Clicking and dragging the resize handle.
+* Focussing the resize handle, and using the arrow keys. The **Up Arrow** and **Left Arrow** keys will make the editor smalller, and the **Down Arrow** and **Right Arrow** keys will make the editor larger.
```
Comment (lnewson) [inline _includes/configuration/resize.md:8]:
```suggestion
* Focussing the resize handle, and using the arrow keys. The **Up Arrow** and **Left Arrow** keys will make the editor smaller, and the **Down Arrow** and **Right Arrow** keys will make the editor larger.
```

File: _includes/configuration/resize.md [hunk 3]
```diff
@@ -2,6 +2,11 @@
 
 This option gives you the ability to disable the resize handle or set it to resize both horizontal and vertically. The option can be true, false or the string `'both'`. `False` disables the resize, `true` enables vertical resizing only, `'both'` makes it possible to resize in both directions horizontal and vertical.
 
+When resizing is enabled, the editor can be resized by either:
+
+* Clicking and dragging the resize handle.
```
Comment (lnewson) [inline _includes/configuration/resize.md:7]:
```suggestion
* Clicking and dragging the resize handle in the bottom right-hand corner.
```
Maybe?

**Review by lnewson — APPROVED** [review]

File: _includes/configuration/resize.md (hunk 3 above)
Comment (lnewson) [reply _includes/configuration/resize.md:7]: It's probably not needed though and if it is, it should probably be in the first sentence for this setting.

File: _includes/configuration/resize.md [hunk 4]
```diff
@@ -2,6 +2,11 @@
...
+* Focussing the resize handle, and using the arrow keys. The **Up Arrow** and **Left Arrow** keys will make the editor smaller, and the **Down Arrow** and **Right Arrow** keys will make the editor larger.
```
Comment (JamesToohey) [inline _includes/configuration/resize.md:8]:
```suggestion
* Focusing the resize handle, and using the arrow keys. The **Up Arrow** and **Left Arrow** keys will make the editor smaller, and the **Down Arrow** and **Right Arrow** keys will make the editor larger.
```

**Review by JamesToohey — APPROVED** [review]

**Review by tylerkelly13 — APPROVED** [review]

File: _includes/configuration/resize.md [hunk 5]
```diff
@@ -2,6 +2,11 @@
 
 This option gives you the ability to disable the resize handle or set it to resize both horizontal and vertically. The option can be true, false or the string `'both'`. `False` disables the resize, `true` enables vertical resizing only, `'both'` makes it possible to resize in both directions horizontal and vertical.
 
+When resizing is enabled, the editor can be resized by either:
+
+* Clicking and dragging the resize handle in the bottom right-hand corner.
```
Comment (metricjs) [inline _includes/configuration/resize.md:7]:
```suggestion
* Clicking and dragging the resize handle in the bottom right-hand corner of the editor.
```
? redundant, but still...

File: _includes/configuration/resize.md [hunk 6]
```diff
@@ -2,6 +2,11 @@
...
+* Focusing the resize handle, and using the arrow keys. The **Up Arrow** and **Left Arrow** keys will make the editor smaller, and the **Down Arrow** and **Right Arrow** keys will make the editor larger.
```
Comment (metricjs) [inline _includes/configuration/resize.md:8]: I guess I was thinking of something more specific, but also that would be complicated with the different resizing modes, so I guess this works.

**Review by metricjs — APPROVED** [review]

File: _includes/configuration/resize.md (hunk 3 above)
Comment (ashfordneil) [reply _includes/configuration/resize.md]: Making a Jira to consider moving this to the first sentence for this setting, as it's out of scope. For now, I'm going to just accept your suggestion.

**Review by EkimChau — APPROVED** [review]

**Pull request merged**